From: Kevin Locke <kevin@kevinlocke.name>
Date: Mon, 3 Dec 2018 11:41:43 -0700
Subject: Make.Rules: Allow overriding DYNAMIC

So that we can build dynamic binaries even when building from a git
repo (e.g. with git-buildpackage).

Forwarded: not-needed
Last-Update: 2018-12-03
---
 Make.Rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Make.Rules b/Make.Rules
index 5366f2d..4fe52ae 100644
--- a/Make.Rules
+++ b/Make.Rules
@@ -81,7 +81,7 @@ LDFLAGS += -L$(topdir)/libcap
 CFLAGS += -Dlinux $(WARNINGS) $(DEBUG)
 PAM_CAP ?= $(shell if [ -f /usr/include/security/pam_modules.h ]; then echo yes ; else echo no ; fi)
 INDENT := $(shell if [ -n "$$(which indent 2>/dev/null)" ]; then echo "| indent -kr" ; fi)
-DYNAMIC := $(shell if [ ! -d "$(topdir)/.git" ]; then echo yes; fi)
+DYNAMIC ?= $(shell if [ ! -d "$(topdir)/.git" ]; then echo yes; fi)
 
 GO := go
 GOLANG := $(shell if [ -n "$(shell $(GO) version 2>/dev/null)" ]; then echo yes ; else echo no ; fi)
